krita_server.py

```diff
@@ -240,18 +251,23 @@ async def f_img2img(req: Img2ImgRequest):
         req.batch_size or opt['batch_size'],
         req.cfg_scale or opt['cfg_scale'],
         req.denoising_strength or opt['denoising_strength'],
+        None,
         seed,
+        None,
+        0,
+        0,
+        0,
         height,
         width,
         opt['resize_mode'],
         upscaler_index,
         req.upscale_overlap or opt['upscale_overlap'],
         req.inpaint_full_res or opt['inpaint_full_res'],
-        0,
+        req.mask_invert or opt['mask_invert'],
```

Comment:
I'm not sure that will work. The problem is that plugin saves the image with the mask applied already. The server doesn't even get masked part of the image. I think some changes in krita_diff.py are needed for this thing to work. 

However I'm fine if this option ends in the master. It can be fixed later. Probably better to hide it though.